src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -189,8 +189,16 @@ impl<'a, 'tcx> Qualifier<'a, 'tcx, 'tcx> {
 
     /// Error about extra statements in a constant.
     fn statement_like(&mut self) {
+        if self.tcx.sess.features_untracked().const_let {
+            return;
```

Comment:
This seems bad, because `statement_like` is used for a lot of errors - it should maybe be split into several functions?